noetic/distribution.yaml

```diff
@@ -1746,6 +1746,28 @@ repositories:
       url: https://github.com/praveen-palanisamy/multiple-object-tracking-lidar.git
       version: master
     status: maintained
+  multisense_ros:
```

Comment:
@tfoote this PR releases into multiple ROS distros, including Noetic which is currently [holding for an upcoming sync](https://discourse.ros.org/t/preparing-for-noetic-sync-2020-07-03/15171)